src/backend/media-route-handlers.ts

```diff
@@ -90,7 +103,8 @@ export async function mediaPresignedUrl(req: Request, res: Response) {
     Bucket: bucketName,
     ContentType: mimeType,
     Metadata: { 'Content-Type': mimeType },
-    Tagging: `editorVariant=${editorVariant}&editorHost=${req.headers.host}`,
+    // saved as tags so we can potentially use it in IAM policies later:
+    Tagging: `editorVariant=${editorVariant}&editorHost=${editorHost}${userIdTag}`,
```

Comment:
> editorHost=${editorHost}${userIdTag}`

Isn't it easier if we add a separator here? Something like `${editorHost}:${userIdTag}`

Reply:
And a question here: Why isn't it in Metadata?

Reply:
> editorHost=${editorHost}${userIdTag}`
Isn't it easier for a future search if we add a separator here? Something like ${editorHost}:${userIdTag}

`userIdTag` is defined as `&userId=${userId}`

Reply:
> And a question here: Why isn't it in Metadata?

See comment above:
`    // saved as tags so we can potentially use it in IAM policies later:`